rs_scraper.py

```diff
@@ -1,17 +1,16 @@
 import requests, sqlite3
 from lxml import html
 
-class RS_Scraper:
+class RSScraper:
+
     def __init__(self, thread):
         #initialize scraping object
-        self.thread = thread
-        self.timestamps = []
-        self.bodies     = []
-        self.usernames  = []
+        self.thread = thread  #url of the thread to be scraped
 
-        tmp = self.create_tree(thread) #here2
-        self.current_page = self.get_current_page(tmp)
-        self.max_page     = self.get_max_page(tmp)
+        self.__create_tree__("") #create a tree and set it to current page
+        self.max_page     = self.get_max_page()     #total number of pages in the thread
```

Comment:
these two lines probably don't need comments, it's clear enough what's happening here